### ISSUE TITLE
[Effect Suggestion] Dirty Car

### ISSUE THREAD

**Issue by RosyMiranto18** [gta-chaos-mod/ChaosModV#1514]
**Name of effect**
Dirty Car

**Duration if timed effect (long or short timed?)**
One-Time

**Effect Description**
Turned your car to be completely dirty and muddy while every car around you is also dirty. The dirt and mudnon your car cannot be removed until the car is despawned. Did you just go offroading last night?

**Comment by DEATHCON1** [comment]
Have you seen how dirty vehicles got on old gen?

**Comment by RosyMiranto18** [comment]
I have played GTA V back in the PS3 days and i completely forgot. It's been years. But it doesn't matter since Chaos Mod is for PC not Console. Is the dirt more on PS3 compared to PS4 though?

**Comment by DEATHCON1** [comment]
Yes, way more. The whole car from roof to wheel got really dirty like you've just been rallying. Even the windows got equally as dirty as the panels.

**Comment by RosyMiranto18** [comment]
I see. Weird that the older gen is more realistic with the dirth car. Alright then, take the old gen dirt instead of the current gen it is.
